pipeline_standards.md

```diff
@@ -0,0 +1,42 @@
+# Pipeline Best Practices
+
+The range of pipeline projects that the ACF Data Surge team may take on will vary in scope and complexity. This guide foregoes specific recommendations in favor of providing a set of high-level considerations central to a well-executed pipeline project.
+
+## Define clear goals/outcomes 
+Understand and clearly define the business goals by knowing the answers  to the following questions:
+- What is the problem?
+    - Thoroughly understand the purpose of the pipeline. Spend significant time discussing the broad goal of the pipeline with stakeholders so that this goal is always in focus during pipeline development.
+- What data sources/destinations are needed?
+    - Clearly define where data are coming from and where they will ultimately be stored.
+- What transformations should be applied?
+    - Clarify how the data must be manipulated, if at all, en-route to its destination.
+- What outputs are expected?
+    - Define the format of data exiting the pipeline.
+- How will success be measured?
+    - Establish clear metrics for a successful pipeline.
+
+## Choose the right tools and technologies
+Depending on the data type, volume, and velocity, choose appropriate tools and technologies. For example: 
+- Data ingestion tools: Apache Kafka, Logstash, Apache NiFi
+- Data storage: 
+    - SQL Databases (PostgreSQL, MySQL)
+    - NoSQL Databases (MongoDB, Cassandra, DynamoDB):
+    - Data Warehouses (Google BigQuery, Amazon Redshift, Snowflake)
+- Monitoring and logging: Grafana, Elasticsearch, Logstash, and Kibana
+- Data processing frameworks: Apache Spark, Apache Hadoop, Apache Flink, Pandas
+- Data orchestration: Apache Airflow 
+
+## Scalability and flexibility 
+Where possible, design the pipeline to be easily scaled up or down and to adapt to changes in data types and data formats.
+
+## Implement data quality checks
+Data quality is important for any data-driven decision-making. The data pipeline should be able to handle invalid, missing, or inconsistent data. Ensure that the pipeline includes steps for validating data at various stages to catch errors, inconsistencies, or missing data early. Formulate a set of rules for how to handle such issues.
+
+## Monitoring and optimizing
+Continuously monitor the performance of the pipeline and seek opportunities to optimize data processing times, reduce costs, and improve data quality. Implement monitoring and logging to track the performance and health of the pipeline. Alerts should be set up for failures or significant performance degradations. Logs can include assessments of data quality and any major errors or inconsistencies caught during data quality checks.
+
+## Ensure security and compliance
```

Comment:
Let's include a link to the [HHS approved software list](https://hhsgov.sharepoint.com/sites/asa/ocio/ops/ITServiceCentral/Software%20Approval/Forms/AllItems.aspx) (noting that the link is only accessible within HHS).